### ISSUE TITLE
Ads in Firefox addons

### ISSUE THREAD

**Issue by CHEF-KOCH** [gorhill/uBlock#1001]
I'm using this little addon called [IP Address and Domain Informatio](https://addons.mozilla.org/en-US/firefox/addon/ip-address-and-domain-info/) which includes several Ads I currently can't block without manipulating the addon itself. So my question is is it possible to block this via uBlock (1.3.7+) on Firefox 42.0.1?

![unbenannt](https://cloud.githubusercontent.com/assets/727123/11534969/ee3ad4b6-9911-11e5-8564-7b769cc5e2d2.png)

I tried several settings (e.g. blocking 3rd-party scripts/frames) and filter lists but nothing, I see in the logger window utlsapi.com + tcpiputils.com but nothing which points me to the ads, so I guess this is coming directly from _tcpiputils.com_. 

The Ads are changing from time to time, one of them is <code>http://adsclickb.everified.hop.clickbank.net/?item=3&si=20</code> and blocked via EasyPrivacy [||hop.clickbank.net^] (but still visible?).

**Comment by gorhill** [comment]
They are blocked on my side, because:

- I have _advanced user_ mode enabled.
- I removed `behind-the-scene` from the _Whitelist_ pane.
    - Do this **if and only if** you understand all the consequences: I will not provide any support from browser issues arising from removing the whitelisting of `behind-the-scene`.
- I added the dynamic filtering rules `behind-the-scene * 3p noop`, `behind-the-scene * 3p-script noop`, `behind-the-scene * 3p-frame noop`: to let only static filters act for behind-the-scene requests.

Cosmetic filters do not act on extension panels -- the content scripts are not injected in such panels.
